operator/zitadel/kinds/iam/zitadel/ingress/controllers/nginx/adapt_test.go

```diff
@@ -0,0 +1,37 @@
+package nginx
+
+import (
+	"testing"
+
+	"github.com/caos/orbos/mntr"
+	"github.com/caos/orbos/pkg/labels/mocklabels"
+	"github.com/caos/zitadel/operator/zitadel/kinds/iam/zitadel/ingress/protocol/core"
+)
+
+func TestAdaptFuncCover(t *testing.T) {
```

Comment:
Tests with no content? There are no assertions, no mocking, and no use of the functions outside?
We shouldn't include tests that only cover the functions, they are more or less useless, either with assertions or delete altogether.